sycl/source/detail/stream_impl.hpp

```diff
@@ -35,6 +37,8 @@ class __SYCL_EXPORT stream_impl {
   // buffer and offset in the flush buffer
   GlobalOffsetAccessorT accessGlobalOffset(handler &CGH);
 
+  void fill(QueueImplPtr Queue);
```

Comment:
This function is only used within limited use-case. Is it possible to make it static with visibility only in cpp file?

Reply:
Done.